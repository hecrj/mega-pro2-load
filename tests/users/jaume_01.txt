4 

10
30
50
110

3

1 3 0 0 2 0 0

10
4 1 2 3 4
10
3 1 2 3
10
3 1 2 3

-1 3 2

-1 2 3

-5 1 4

-1 1 15

-1 4 50

-1 3 51

-1 2 52

-2 53

-3 54

-6
