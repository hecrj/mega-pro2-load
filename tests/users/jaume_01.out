Peticion procesada y servidores
0 3
1 3
Peticion procesada y servidores
1 3
2
Pelicula mas solicitada
2 1
Peticion procesada y servidores
2 1
1
Peticion procesada y servidores
3 11
1
Peticion procesada y servidores
4 5
3
Peticion procesada y servidores
5 3
2
Peticiones pendientes
5 2 52 55
4 3 51 56
3 4 50 61
Servidores ocupados
1 3 7
2 5 1
3 4 2
